src/js/jsx/sections/libraries/LibraryList.jsx

```diff
@@ -296,6 +307,7 @@ define(function (require, exports, module) {
                     live={false}
                     autoSelect={false}
                     onChange={this._handleChangeLibrary}
+                    onBlur={this._handleLibraryListBlur}
```

Comment:
seems slightly odd that you have to register a handler, just to in turn call a method on itself.  Like, maybe this would also work as an optional property ~ `releaseOnBlur`.  If there's one thing Datalist needs more of, its optional parameters.